include/aca_config.h

```diff
@@ -29,4 +29,8 @@
 
 #define USLEEPTIME_IN_MICROSECONDS 15000
 
```

Comment:
@zzxgzgz If we don't use it anymore, please remove it.

Reply:
Roger